inject/src/main/java/io/micronaut/context/DefaultBeanContext.java

```diff
@@ -1974,7 +1974,7 @@ protected void initializeContext(
                     initializeEagerBean(eagerInitDefinition);
                 } catch (DisabledBeanException e) {
                     if (AbstractBeanContextConditional.ConditionLog.LOG.isDebugEnabled()) {
-                        AbstractBeanContextConditional.ConditionLog.LOG.debug("Bean of type [{}] disabled for reason: {}", eagerInitDefinition.getBeanType().getSimpleName(), e.getMessage());
+                        AbstractBeanContextConditional.ConditionLog.LOG.debug("Bean of type [{}] disabled for reason: {}", eagerInitDefinition.getBeanType().getSimpleName(), e.getMessage(), e);
```

Comment:
Stacktrace unlikely to add value here.
```suggestion
                        AbstractBeanContextConditional.ConditionLog.LOG.debug("Bean of type [{}] disabled for reason: {}", eagerInitDefinition.getBeanType().getSimpleName(), e.getMessage(), e);
```